modules/test-tools/src/main/kotlin/jp/co/soramitsu/iroha2/testengine/IrohaContainer.kt

```diff
@@ -139,11 +148,12 @@ open class IrohaContainer : GenericContainer<IrohaContainer> {
 
     companion object {
         const val NETWORK_ALIAS = "iroha"
-        const val DEFAULT_IMAGE_TAG = "lts@sha256:4072d09bee376c8fcbc586d35ea25624738694ce2c37aa0db60c9998c613048b"
-        const val DEFAULT_IMAGE_NAME = "hyperledger/iroha2"
+        const val DEFAULT_IMAGE_TAG = "sha256:4b61b866e15039989e689e3403986ebb207628c36d956d2dc99078e19764e212"
```

Comment:
```suggestion
        const val DEFAULT_IMAGE_TAG = "lts-b783f10fa7de26ed1fdd4c526bd162f8636f1a65"
```
you might try this, without explicit hash reference